verification/verification.go

```diff
@@ -59,6 +65,8 @@ var AllTestCases = []TestCase{
 	CertifyKeySimulationTestCase,
 	GetCertificateChainTestCase,
 	TagTCITestCase,
+	ExtendTCITestCase,
+	ExtendDerivedTciTestCase,
```

Comment:
Reverted the testcases order in this list, added newly created testcases (that were split from TagTCI testcase)